### PR TITLE
TILA-631

### PR DESCRIPTION
**Pull request by siren**
- generate resolution pdf
- added download link to application page

### CODE REVIEW

File: src/applications/Reservations.tsx [hunk 1]
```diff
@@ -213,152 +244,215 @@ const Reservations = (): JSX.Element | null => {
...
-                            <div>
-                              {' '}
-                              {t('common.time', {
-                                date: begin,
-                              })}{' '}
-                              - {t('common.time', { date: end })}
-                            </div>
-                          </>
+        {reservations.data?.length === 0 ? (
+          <span> no reservations</span>
```
Comment (kieferbonk) [inline src/applications/Reservations.tsx:292]: Translation needed?

File: src/applications/Reservations.tsx [hunk 2]
```diff
@@ -213,152 +244,215 @@ const Reservations = (): JSX.Element | null => {
...
+                              <>
+                                <HorisontalRule />
+                                <TwoColLayout>
+                                  <IconHome />
+                                  <ReservationUnitName>
+                                    {reservationUnit.name.fi}
+                                  </ReservationUnitName>
+                                  <IconInfoCircle />
+                                  <ContactInfo>
+                                    TODO contact info from API
```
Comment (kieferbonk) [inline src/applications/Reservations.tsx:415]: Done later?

File: src/i18n/messages.ts [hunk 3]
```diff
@@ -486,9 +486,18 @@ const translations: Translations = {
   Reservations: {
     back: ['Takaisin Omat hakemukset'],
     info: ['Hakemukselle on myönnetty seuraavat vakiovuorot'],
+    titleResolution: ['Vuorojaon päätös'],
+    resultWithReservations: [
+      'Hakemuksellesi on myönnetty seuraavat käyttövuorot.',
+    ],
+    resultWithoutReservations: [
+      'Hakemuksellesi ei valitettavasti ole voity myöntää käyttövuoroja.',
```
Comment (kieferbonk) [inline src/i18n/messages.ts:494]: Typo

**Review by kieferbonk — APPROVED** [review]

File: src/applications/Reservations.tsx (hunk 2 above)
Comment (siren) [reply src/applications/Reservations.tsx:415]: yeah, different issue, will submit pr after this is merged

File: src/applications/Reservations.tsx (hunk 1 above)
Comment (siren) [reply src/applications/Reservations.tsx:292]: will fix this in later issue

**Pull request merged**